services/viva-ms/src/helpers/dynamoDb.js

```diff
@@ -55,3 +58,79 @@ export function updateCaseExpirationTime(caseUpdateParams) {
 
   return dynamoDb.call('update', updateParams);
 }
+
+export function getCaseListOnPeriod(vivaPerson) {
```

Comment:
**OPTIONAL**

... Another thought.

Why don't we have `personalNumber`, `startDate` and `endDate` as the only parameters instead?

example:

```js
export function getCaseListForPeriod({ personalNumber, startDate, endDate }) {
  const casesQueryParams = {
    TableName: config.cases.tableName,
    KeyConditionExpression: 'PK = :pk',
    FilterExpression:
      'details.period.startDate = :periodStartDate AND details.period.endDate = :periodEndDate',
    ExpressionAttributeValues: {
      ':pk': `USER#${personalNumber}`,
      ':periodStartDate': startDate,
      ':periodEndDate': endDate,
    },
  };

  return dynamoDb.call('query', casesQueryParams);
}
```

Which gives us the following benefits:
1. The functions gets "dumber" and do one thing only
2. The parameters align more with the function name i.e. startDate and endDate. startDate and endDate defines a period.